### DISCUSSION TITLE
Tether/harpoon turret. syringe turrets.

### DISCUSSION THREAD

**Discussion by Psyduckdeluxe** [FakeFishGames/Barotrauma#10168] [Ideas and suggestions]
Was really trying to make a harpoon gun so i could hook monsters and people.
So tagging the hook gun in to a turret and it does shoot once and all. But no way to reload its sadly!
Same for syringe gun! used to be pretty cool to make a syringe drone! 

So i guess what i am asking if we can have a turret that could potential equip the hand weapons?! 
and possible reload them! 

Also can we please have the sprite level jump be less than .100
![image](https://user-images.githubusercontent.com/78105038/198366101-dc9b7f15-bdae-45c4-96ca-1832226ac1e6.png)

happy gaming!

**Comment by Ek-Videogames** [comment]
As a goof I tried implementing it as a mod when ropes were first added to the game, turned out the physics engine doesn't know what to do when you rope 2 submarines together so it freaks out. _(Since normally thalamus' are static, only applying force to the submarine)_

So it would need some tweaking to make the player submarine _(and other attached submarine in PVP / Pirate mission)_ experience reaction forces correctly.

**Reply by Psyduckdeluxe** [reply]
Ahh that sucks. You know if its the same for the hook gun on creatures ?
